app/pages/search/results/SearchResults.js

```diff
@@ -12,6 +12,8 @@ import searchResultsSelector from './searchResultsSelector';
 
 export class UnconnectedSearchResults extends Component {
   componentDidMount() {
+    // TODO: fix this lint
+    // eslint-disable-next-line react/no-find-dom-node
     scrollTo(findDOMNode(this));
```

Comment:
We gonna replace this with `React.createRef` when we upgrade React to higher version.

Reply:
Why don't we use ref callbacks for now, and replace with the new API later on, instead of findDOMNode?